app.rb

```diff
@@ -0,0 +1,8 @@
+require 'sinatra/base'
+
+class Bowling < Sinatra::Base
+  get "/" do
+    File.read('public/index.html')
```

Comment:
Very cool that you've used Ruby to serve this up. This should allow you to easily add functionality to save state, introduce highscores etc.